gamification/src/index.js

```diff
@@ -69,6 +71,9 @@ ReactDOM.render(
           </Route>
           <Route path="/addstudent">
             <AddStudent />
+          </Route>
```

Comment:
refactor to subjects no need to add teacher sufix

Reply:
url has to be called subjects
Refactor component
WIP